Source/WebKit/WebProcess/GPU/graphics/RemoteGraphicsContextGLProxy.messages.in

```diff
@@ -23,7 +23,7 @@
 #if ENABLE(GPU_PROCESS) && ENABLE(WEBGL)
 
 messages -> RemoteGraphicsContextGLProxy NotRefCounted {
-    void WasCreated(bool didSucceed, IPC::Semaphore streamWakeUpSemaphore, IPC::Semaphore streamClientWaitSemaphore, String availableExtensions, String requestableExtensions)
+    void WasCreated(bool didSucceed, IPC::Semaphore streamWakeUpSemaphore, IPC::Semaphore streamClientWaitSemaphore, WebKit::RemoteGraphicsContextGLInitializationState initializationState)
```

Comment:
As a optional (pun intended) beautification, you might replace `didSucceed` with `std::optional<WebKit::RemoteGraphicsContextGLInitializationState>`